scripts/chpass.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+"""Change user password
+
+Usage:
+    chpass.py <user_email> <stage> <api_token> [options]
+
+    --unlock  If set, unlocks user account after changing the password
+
+Example:
+    ./chpass.py admin@example.com dev myToken --unlock
+"""
+
+import sys
+import getpass
+import logging
+
+logger = logging.getLogger('chpass')
+logging.basicConfig(level=logging.INFO)
+
+from docopt import docopt
+
+from dmutils import apiclient
+
+
+def get_api_endpoint_from_stage(stage):
+    stage_prefixes = {
+        'preview': 'preview-api.development',
+        'staging': 'staging-api',
+        'production': 'api'
+    }
+
+    if stage in ['local', 'dev', 'development']:
+        return 'http://localhost:5000'
+
+    return "https://{}.digitalmarketplace.service.gov.uk".format(
+        stage_prefixes[stage]
+    )
+
+
+def update_user_password(api_endpoint, api_token, user_email, new_password, unlock=False):
+    data_client = apiclient.DataAPIClient(api_endpoint, api_token)
+
+    try:
+        user = data_client.get_user(email_address=user_email)['users']
+        data_client.update_user_password(user['id'], new_password)
+        if unlock:
+            data_client.update_user(user['id'], locked=False)
+            logger.info("User unlocked")
+    except apiclient.APIError:
+        sys.exit(1)
```

Comment:
Logging is configured for the script run, so it prints `apiclient` warnings with status codes and error messages